src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java

```diff
@@ -54,7 +54,10 @@
         extends AbstractReferenceCounted
         implements Bytes<U>,
         HasUncheckedRandomDataInput {
-    protected static final boolean DISABLE_THREAD_SAFETY = Jvm.getBoolean("disable.thread.safety", false);
+    protected static final boolean DISABLE_SINGLE_THREADED_CHECK =
+            Jvm.getBoolean("disable.single.threaded.check",
+                    Jvm.getBoolean("disable.thread.safety", false));
+    protected static final boolean DISABLE_THREAD_SAFETY = DISABLE_SINGLE_THREADED_CHECK;
```

Comment:
Are we going to phase out `disable.thread.safety` in the future? should we warn if we detect people using it?

Reply:
Also I wonder if we should expose the value from AbstractCloseable and defer to that? then we could just have it detected/set in one place?

Reply:
e.g. `AbstractCloseable.areThreadSafetyChecksDisabled()`

Reply:
Or even `Jvm.areThreadSafetyChecksDisabled()` which AbstractCloseable and this both defer to?

Reply:
Jvm or SingleThreadedChecked methods would be best.